### ISSUE TITLE
Question: Available methods for leveraging the latest changes

### ISSUE THREAD

**Issue by ericmacfa** [fastify/fastify-oauth2#23]
As the title states, I'm looking for the best way to leverage the latest changes.  
Is there anything like an `@next` dist-tag that can be leveraged, or does the code have to be built manually?

**Comment by Eomm** [comment]
No, the next dist tag isn't applied to plugins.

Anyway, I'm waiting for a review of an open PR to release a new version with:
```
📚 PR:
- fix: Typo in method name (#22)
- feature: add callback uri params (#19)
```

I could plan to release later this evening, it takes 5 min to me if you need this feature 👍

**Comment by ericmacfa** [comment]
Understood 👍thanks for the info!
I'm most interested in #19, but my need isn't urgent, so whenever works best for you!
